recipes/winpcap/all/conanfile.py

```diff
@@ -0,0 +1,29 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class WinpcapConan(ConanFile):
+    name = "winpcap"
+    version = "4.1.2"
```

Comment:
Don't hardcode the version, remove it and use something like `conan create . winpcap/4.1.2@bincrafters/stable` to specify the version locally

Reply:
Thanks, done! 👍